### ISSUE TITLE
Move or eliminate contents menu in print and mobile style

### ISSUE THREAD

**Issue by ausmeyer** [PeerJ/paper-now#49]
The print and mobile stylesheets have the contents section between the normal sections and the figures.  In mobile, it seems like it should be either before or immediately following the abstract. In print, it seems like it should be removed altogether.

Either way, the current position sort of defeats its function.

**Comment by hubgit** [comment]
Good point, the contents probably shouldn't be there in the mobile view either - I'll remove it (and maybe restore it in a hidden menu at some point).
